recipes/rastervision-pipeline/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "rastervision-pipeline" %}
+{% set version = "0.13.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/rastervision_pipeline-{{ version }}.tar.gz
+  sha256: 2f4a4bce099aeeb1c59acd948fb1209fce8bbdde58cc2469e2cd9a169e337a95
+  patches:
+    - 0001-fix-imports.patch
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - rastervision=rastervision.pipeline.cli:main
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - click ==6.*
+    - everett ==0.9
+    - pydantic ==1.2
+    - python >=3.6
+    - six ==1.11.*
+    - typing-extensions ==3.7.4.1
+    - urllib3
+    - brotlipy
+
+test:
+  imports:
+    - rastervision
```

Comment:
Is `rastervision` even a useful import for this package to actually verify it's usaable? What about `import rastervision.pipeline`?

Reply:
Good call